.github/ISSUE_TEMPLATE/5-bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: (Advanced) Bug report
```

Comment:
I'd personally keep the html commented disclaimer that is at the top of the current templates.